src/tests/unit/tests/DetailsView/components/save-assessment-button.test.tsx

```diff
@@ -3,11 +3,16 @@
 import * as React from 'react';
 import { shallow } from 'enzyme';
 
-import { SaveAssessmentButton } from 'DetailsView/components/save-assessment-button';
+import {
+    SaveAssessmentButton,
+    SaveAssessmentButtonProps,
+} from 'DetailsView/components/save-assessment-button';
 
 describe('SaveAssessmentButton', () => {
+    let props: SaveAssessmentButtonProps;
+
     it('should render per the snapshot', () => {
-        const rendered = shallow(<SaveAssessmentButton />);
+        const rendered = shallow(<SaveAssessmentButton {...props} />);
```

Comment:
You probably want to set some test values for `props.download` and `.href` so that the snapshot for this test shows those properties getting set in the rendered `InsightsCommandButton`